HelloTorchIOS/HelloTorchIOS/StrobeLights.swift

```diff
@@ -1,9 +1,39 @@
-//
-//  StrobeLights.swift
-//  HelloTorchIOS
-//
-//  Created by Luis Valle on 02/05/2018.
-//  Copyright © 2018 Novoda. All rights reserved.
-//
-
 import Foundation
+import AVFoundation
+
+class StrobeLights {
+    
+    let lightsController = LightsLogic()
```

Comment:
Controller is reserved on IOS for `ViewControllers` might be better to rename this variable